### ISSUE TITLE
Provide keyboard navigation for Clarity tabs

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#3971]
## Summary

User should be able to navigate to next/previous tab with left/right arrow keys

* What is the change?
When a tab header is focused user should be able to navigate to next tab with right arrow key ant to the previous one with the left arrow key.
* Why should it go in Clarity?
It is a basic requirement for accessibility compliance - https://www.w3.org/TR/WCAG20/#keyboard-operation
* Does this change impact existing behaviors? If so how?
No 
* If this change introduces a new behavior, is this behavior accessible?
Yes
## Use case

when I am navigating through tabs I should be able to do it with arrow keys of the keyboard

## Examples

http://whatsock.com/tsg/Coding%20Arena/ARIA%20and%20Non-ARIA%20Tabs/ARIA%20Tabs%20(Internal%20Content)/demo.htm
## Workarounds
No

**Comment by gnomeontherun** [comment]
This is already supported when using Angular component for tabs. Static (HTML/CSS) tabs don't have this functionality and if you need it please use the Angular component.

Closing as already resolved and duplicate of #3502.
